content/docs/quickstart/kubernetes/_index.md

```diff
@@ -6,12 +6,32 @@ menu:
     identifier: kubernetes
     name: Kubernetes
     weight: 2
+cloudlink: /docs/reference/clouds/kubernetes
 ---
+Learn how to quickly provision infrastructure on Kubernetes with Pulumi using JavaScript, TypeScript, or Python as your language runtime.
 
-This guide is intended to help you quickly get started provisioning infrastructure on Kubernetes with Pulumi.
+Pulumi's Cloud Native SDK makes it easy to target any Kubernetes environment to provision a cluster, configure and deploy applications, and update them as required.
 
-<!-- TODO embellish the summary/overview of the guide. -->
+Pulumi works with Minikube, on-premises and cloud-hosted custom Kubernetes clusters, and the managed services from Google (GKE), Azure (AKS), and Amazon (EKS).
+
+<div class="bg-transparent border border-gray-300 hover:bg-gray-200 mb-5 flex justify-center">
```

Comment:
I see this block renders some icons and defines some hover behavior, but I'm not quite understanding what the intent is. What's it for?

Reply:
I took the icons from https://www.pulumi.com/kubernetes/ and added the Pulumi icon, as the suggested addition of moving the diagram from https://www.pulumi.com/docs/reference/clouds/kubernetes/ to the Kubernetes quickstart intro just makes it look cluttered. Open to your ideas though -- maybe we can have a simple diagram on how Pulumi works with different Kubernetes environments + the languages you can use in the guide.

Reply:
Ah, ok -- and totally agree that diagram is way more than you'd want to see at this stage. 😄 

I think I understand, and agree we could likely do meet the need with some some light visual treatment and maybe a few accompanying words. I'll think a bit on it and maybe submit something separately for you to have a look at. Sound ok?

Reply:
Yes, sounds excellent. Thanks!